src/App.js

```diff
@@ -1,5 +1,40 @@
-function App() {
-  return <div className="App"></div>
+import { useState, useEffect } from 'react'
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+export default function App() {
+  const [tracks, setTracks] = useState([])
+
+  useEffect(() => getTracks(), [])
+
+  function getTracks() {
+    fetch('http://wandergold.local/track')
+      .then((res) => res.json())
+      .then((data) => setTracks(data))
+      .catch((error) => console.error('Error:', error))
+  }
+
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/">
+          {tracks.map(({ id, description, title }) => (
+            <Track key={id}>
+              <p>{title}</p>
+              <p>{description}</p>
+            </Track>
+          ))}
+        </Route>
+        <Route exact path="/signup">
+          <div>Register</div>
+        </Route>
+      </Switch>
+    </Router>
+  )
 }
 
-export default App
+const Track = styled.section`
+  width: 300px;
+  height: 150px;
+  background: #eee;
+`
```

Comment:
Is there a reason, you do not have a separate folder for your frontend? I think we discussed that this would be best practice.

Reply:
Hm, if this is indeed a problem, it would be serious. I understood it to be an option to just put the backend into a subfolder. Reason is complexity-reducing.

Reply:
Moved files to /frontend